src/api.ts

```
@@ -0,0 +1,159 @@
+        },
+    });
+    if (response.status !== 200) {
+        throw new Error(response.statusText);
+    }
+    const result = await response.json() as CardResponse;
+    return result.data.cards;
+}
+interface EnvResponse {
+    "data": {
+        "result": {
+            "variables": {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)